snuba/clickhouse/astquery.py

```diff
@@ -46,7 +43,7 @@ def __init__(self, query: Query, settings: RequestSettings,) -> None:
         self.__settings = settings
         self.__sql_data_list: Optional[Sequence[Tuple[str, str]]] = None
         self.__formatted_query: Optional[str] = None
-        self.__sql_data: Optional[Sequence[str, str]] = None
```

Comment:
Was this just incorrect before? It doesn't seem like this changed.

Reply:
It was wrong before.